contrib/raftexample/raft.go

```diff
@@ -77,27 +85,143 @@ func newRaftNode(id int, peers []string, join bool, proposeC <-chan string,
 		peers:       peers,
 		join:        join,
 		waldir:      fmt.Sprintf("raftexample-%d", id),
+		snapDir:     fmt.Sprintf("raftexample-%d-snap", id),
 		raftStorage: raft.NewMemoryStorage(),
+		kv:          kv,
```

Comment:
@heyitsanthony Right... should we just create separate channel to send snapshot then? I am just trying to trigger `createSnapshot` method with `kvstore`, which is coupled with raft Ready channel.